scripts/execwhacker.py

```diff
@@ -96,6 +115,24 @@ def process_event(
             logging.exception(e)
 
 
+def check_existing_processes(banned_strings_automaton, allowed_patterns, interval):
+    while True:
+        logging.info("Checking existing processes")
+        count = 0
+        for proc in process_iter():
+            if proc.exe():
+                if kill_if_needed(
+                    banned_strings_automaton,
+                    allowed_patterns,
+                    join(proc.cmdline()),
+                    proc.pid,
+                    ProcessSource.SCAN,
+                ):
+                    count += 1
+        logging.info(f"Killed {count} existing processes")
```

Comment:
Let's keep it in the style of the other log messages, with a `source:`, `action:` etc to make long term grepping easier?